_maps/map_files/Shrike/job_changes.dm

```diff
@@ -98,6 +98,11 @@ MAP_REMOVE_JOB(air_traffic_controller)
 		return
 	hidden = FALSE
 
+// Lathe storage
+/obj/machinery/rnd/production/New()
+	. = ..()
+	base_storage *= 5
```

Comment:
May as well have an actual ore sile and ORM with this modifier combined with the infinite storage in the smelter.

Reduce the multiplier to 2x max.

Reply:
Does the smelter in this PR not have buttons for dispensing exact materials on demand?
Why would you want to move it to an additional device?

Reply:
5 does seem like a lot, we could always just make it *2 and scale it up if that's not enough